### ISSUE TITLE
Allow _type to specifiy the type ...

### ISSUE THREAD

**Issue by lpatiny** [NPellet/visualizer#493]
Currently when we extend types we use the variable name "type"

{type:"array", value: [1,2,3]}

This prevents us of course to use the variable type for another purpose and this as been the source of some bugs ...

In a first step we should allow "_type" as a possibility of name instead of "type" and the data generator should use "_type".

In a second step we could think about removing "type" but this seems not in a close future

**Comment by targos** [comment]
`type` is supposed to be considered special only if present with `value` or `url`. Did you find a place where it was not ?

If we decide to move to `_type`, I think we should do the same with other special properties -> `_value` and `_url`

**Comment by lpatiny** [comment]
You are right. We could at least allow it currently and propose it as the correct way to encode the specific types
